src/main/java/io/fabric8/vertx/maven/plugin/mojos/AbstractRunMojo.java

```diff
@@ -312,8 +310,9 @@ protected void scanAndLoadConfigs() throws MojoExecutionException {
         if (Files.exists(confBaseDir) && Files.isDirectory(confBaseDir)) {
 
             DirectoryScanner directoryScanner = new DirectoryScanner();
-            directoryScanner.setBasedir(this.project.getBasedir() + "/src/main/conf");
-            directoryScanner.setIncludes(new String[]{"*.json", "*.yml", "*.yaml"});
+            directoryScanner.setBasedir(this.project.getBasedir() + Constants.DEFAULT_CONF_DIR);
+            directoryScanner.setIncludes(new String[]{Constants.WILDCARD_JSON_FILES,
```

Comment:
I would only add a String array constant `DEFAULT_INCLUDES`, which holds these patterns literally as array. This is more flexible and reduces some boilerplate.